src/economy.cpp

```diff
@@ -610,7 +610,7 @@ static void CompanyCheckBankrupt(Company *c)
 			Money val = CalculateCompanyValue(c, false);
 
 			c->bankrupt_value = val;
-			c->bankrupt_asked = 1 << c->index; // Don't ask the owner
+			c->bankrupt_asked.Set(c->index); // Don't ask the owner
```

Comment:
Is the behaviour change from setting only 1 bit to adding 1 bit ok here?

Reply:
Good point. I guess it resets it, so lets do that too in the new code.